blog/_posts/2018-10-03-scaladoc-tables.md

```diff
@@ -0,0 +1,103 @@
+---
+layout: blog-detail
+post-type: blog
+by: Janek Bogucki
+title: "Introducing Scaladoc Tables"
+---
+
+Scala 2.12.7 adds support for markdown tables in Scaladoc comments.
```

Comment:
Will it be added in 2.13?  Library authors who cross-compile on 2.11 should avoid for now?

Reply:
Library authors should only publish 2.12 docs or later?

Reply:
It's been merged to 2.13.x IIRC. I feel advising not to use a feature that is only present from 2.12 onwards when cross compiling to 2.11 is overly prescriptive so I vote to not mention this.

Reply:
Agree, doesn't seem like there's a reason to dissuade library authors, but maybe just mention that "library authors who use the table syntax will need to only publish from 2.12"?

Reply:
> It's been merged to 2.13.x IIRC

(it has, but after M5 was released)

Reply:
@ashawley I mentioned the graceful degradation forward compatibility with 2.11 in a new section.